lib/govuk_component/table_helpers.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
```

Comment:
Helper needs drying out.

Reply:
Hi, @naseberry can this PR be closed or are you still intending to get it in?